Makefile

```diff
@@ -15,10 +15,20 @@ LICENSE_IGNORE := -e internal/testdata/
 GO ?= go
 ARGS ?= --strict_message --strict_error
 GOLANGCI_LINT_VERSION ?= v2.4.0
-# Set to use a different version of protovalidate-conformance.
-# Should be kept in sync with the version referenced in buf.yaml and
-# 'buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go' in go.mod.
-CONFORMANCE_VERSION ?= v1.1.0
```

Comment:
Can we add a `target` like this...
```
.PHONY: assert-release-version
assert-release-version:
	@# Used in PRs to main to make sure we don't merge a non-release version.
	test $(PROTOVALIDATE_RELEASE_VERSION) = TRUE
```

And then add a workflow triggered on PRs to `main` that runs it?

Reply:
We intend to merge this as is i.e. always use latest main. Doing it all in one PR will not allow us to work on individual updates to protovalidate.